packages/react-core/src/components/Icon/examples/IconContentSizes.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Icon } from '@patternfly/react-core';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+export const IconContentSizes: React.FunctionComponent = () => (
+  <React.Fragment>
+    <Icon size="xl" iconSize="sm">
+      <PlusCircleIcon />
+    </Icon>
+    <Icon size="xl" iconSize="md">
+      <PlusCircleIcon />
+    </Icon>
+    <Icon size="xl" iconSize="lg">
+      <PlusCircleIcon />
+    </Icon>
+    <Icon size="xl" iconSize="xl">
```

Comment:
This is fine tho it's worth noting that it's redundant, not sure if it would be better to just set `size` to show best practice for when `size` and `iconSize` are the same?